components/camel-cxf/src/main/java/org/apache/camel/component/cxf/jaxrs/CxfRsProducer.java

```diff
@@ -164,6 +164,12 @@ protected void invokeAsyncHttpClient(Exchange exchange, final AsyncCallback call
         setupClientMatrix(client, exchange);
         setupClientQueryAndHeaders(client, exchange);
 
+        // ensure the CONTENT_TYPE header can be retrieved
```

Comment:
We have the `null` on the other side as out standard with the Apache Camel code. Do you mind flipping them around.